src/main/java/com/ca/apim/gateway/cagatewayconfig/util/xml/DocumentTools.java

```diff
@@ -30,6 +30,18 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.util.Map'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=62&issues=AWVkDSPh4Bfu85TPS_PL&open=AWVkDSPh4Bfu85TPS_PL)

[ISSUE_KEY]: AWVkDSPh4Bfu85TPS_PL